MekHQ/src/mekhq/campaign/mission/BotForce.java

```diff
@@ -231,7 +232,14 @@ public void writeToXml(PrintWriter pw1, int indent) {
         if (!getCamouflage().hasDefaultFilename()) {
             MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "camoFileName", getCamouflage().getFilename());
         }
-        MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "colour", getColour().name());
+        
+        // if we've got a legitimate color, great. Otherwise, write out something default
+        if (getColour() != null) {
+            MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "colour", getColour().name());
+        } else {
+            MekHqXmlUtil.writeSimpleXmlTag(pw1, indent, "colour", PlayerColour.FIRE_BRICK.name());
+        }
+        
```

Comment:
This can be removed and replaced with just the standard colour write

Reply:
Still valid, as colour should be back to a non-nullable state.

Reply:
I never really figured out where the original null came from, the parameterless constructor thing was just a guess, so I'm still a little paranoid about it and would prefer to keep the default. This way, if we see a camo turn to fire brick, we know there's another hole somewhere.

Reply:
In that case... can we add an error message with it? That way we can tell within a version or two if that was the cause or not.

Reply:
Will do.